projects/plugins/jetpack/extensions/blocks/tiled-gallery/utils/index.js

```diff
@@ -31,7 +35,7 @@ export function photonizedImgProps( img, galleryAtts = {} ) {
 		return { src: img.url };
 	}
 
-	// Do not Photonize images that are still uploading, are from localhost, or are private + atomic
+	// Do not Photonize images that are still uploading, are from localhost, or are private on simple or WoA sites.
```

Comment:
As per comment here, do we need to include an `isSimpleSite() && isPrivateSite()` check in the if below?

Reply:
I had done that at first. However, since we had the existing logic of `photonWpcomImage` I ended up moving the logic to later in the process. I'll revert that comment change.
dde5f29ef37aa78f4b0d7d2e404174d03ba3e4b3

Reply:
Oh, I see. It makes sense. Although it raises one more question for me. Adding it in the corresponding line and resolving this one.